spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -31,9 +39,35 @@ case class ColumnName(columnName: String, alias: Option[String] = None) extends
   override def selectedAs = alias.getOrElse(columnName)
   override def toString: String = columnName
 
+  def overwrite() = CollectionColumnName(columnName, alias, CollectionOverwrite)
+  def add() = CollectionColumnName(columnName, alias, CollectionAppend)
+  def append() = CollectionColumnName(columnName, alias, CollectionAppend)
+  def prepend() = CollectionColumnName(columnName, alias, CollectionPrepend)
+  def remove() = CollectionColumnName(columnName, alias, CollectionRemove)
```

Comment:
I like the single word better myself because in my mind the "forMethod" ends up reading a little strangely.
`"Col1".forAppending` would actually be appending to Col1 not using Col1 to append something